src/screens/LoginScreen/index.js

```diff
@@ -29,7 +29,9 @@ const LoginScreen = () => {
                         Username:
                         <input className={"form-control m-1"} type={"text"}
                                onChange={(e) =>
-                                   setUserCredential({...userCredential, username: e.target.value})}/>
+                                   setUserCredential({...userCredential, username: e.target.value})}
+                               placeholder="username"
```

Comment:
I feel it's a bit odd to add a "username" placeholder when the label says the same.